styles/stylesheet.css

```diff
@@ -0,0 +1,135 @@
+z* {
+  border: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+html[Attributes Style] {
+    -webkit-locale: "en";
+    direction: ltr;
+    unicode-bidi: isolate;
+}
+
+html {
+  display: block;
+}
+
+body {
+    font-family: "Arial", sans-serif;
+    max-width: 100%;
+    display: block;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  margin: 0;
+  flex-shrink: 0;
+  flex-wrap: wrap;
+  padding: 3em 4em 0;
+  background-image: url(https://imgur.com/364pi2q.png);
+  background-size: cover;
+  color: #FFFFFF;
+  height: 36em;
+}
+
+.headertext {
+  font-size: 1.9rem;
+  font-style: italic;
+  color: green;
+}
+
+.checkoutbox {
+  margin: 0;
+  display: block;
+}
+
+.logoimg {
+  width: 35em;
+}
+
+.cartimg {
+  width: 2em;
+  margin-right: .5rem;
+}
+
+.checkoutbutton {
+    background-color: green;
+    color: #FFFFFF;
+    height: 3em;
+    width: 100%;
+    padding: 1em 2em;
+    border-radius: 0.2em;
+}
+
+/* NAVIGATION */
+
+nav {
+  background-color: #d1d1d1;
+  height: 3.9rem;
+}
+
+nav ul {
+  padding: 18px 0;
```

Comment:
just for consistency, keep to rems for padding/margins to stay with your font sizes. Just a little cleaner.